.cargo/config.toml

```
@@ -1,3 +1,9 @@
 [env]
 RSTEST_TIMEOUT = "300"
 RUST_TEST_THREADS = "1"
+
+[target.x86_64-pc-windows-gnu]
+rustflags = ["-C", "target-feature=+crt-static"]
+
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "target-feature=+crt-static"]
```

I'm not sure if statically linking the C runtime for `x86_64-pc-windows-gnu` is the correct choice (If you're already building using something like MSYS2, you'll probably also run the program inside that environment).
```suggestion
[target.x86_64-pc-windows-msvc]
rustflags = ["-C", "target-feature=+crt-static"]
```

Alternatively, if you want to statically link for `x86_64-pc-windows-gnu` as well, this can be shortened:
```suggestion
[target.'cfg(windows)']
rustflags = ["-C", "target-feature=+crt-static"]
```

---

Hmm, I'm also not sure, but given that the binary size barely increases when statically linking the runtime (and I'm assuming there wouldn't be any other drawbacks?), I think I'll keep it in here. Using `cfg(windows)` is a good idea though, I'll apply that.